axum/src/docs/routing/nest.md

```diff
@@ -82,6 +82,11 @@ let app = Router::new()
 # let _: Router = app;
 ```
 
+Additionally, while the wildcard route `/foo/*rest` will not match the
+paths `/foo` or `/foo/`, a nested router at `/foo` will match the path `/foo`
```

Comment:
Are you sure about the wildcard? `/foo/` should match and I'm pretty sure it does at least in older versions. `rest` will match an empty string but that's allowed.

Reply:
The docs [explicit say this doesn’t happen](https://docs.rs/axum/latest/axum/struct.Router.html#wildcards):

 >Note that `/*key` doesn't match empty segments. Thus:
> 
> - `/*key` doesn't match `/` but does match `/a`, `/a/`, etc.
> - `/x/*key` doesn't match `/x` or `/x/` but does match `/x/a`, `/x/a/`, etc.

Reply:
Right, my bad then. I must have misremembered.